Packs/CortexXDR/Playbooks/Cortex_XDR_-_quarantine_file_v2.yml

```diff
@@ -0,0 +1,245 @@
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "5":
+    id: "5"
+    taskid: 95f7c67b-1956-4f6c-8455-8549fa1b1d86
+    type: regular
+    task:
+      id: 95f7c67b-1956-4f6c-8455-8549fa1b1d86
+      version: -1
+      name: Get quarantine status
+      description: Retrieves the quarantine status for a selected file.
+      script: '|||xdr-get-quarantine-status'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "3"
+    scriptarguments:
+      endpoint_id:
+        complex:
+          root: inputs.endpoint_id
+      file_hash:
+        complex:
+          root: inputs.file_hash
+      file_path:
+        complex:
+          root: inputs.file_path
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 560
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "6":
+    id: "6"
+    taskid: 30c0aca5-fc22-4bea-8523-c53a065fb330
+    type: condition
+    task:
+      id: 30c0aca5-fc22-4bea-8523-c53a065fb330
+      version: -1
+      name: Are the inputs were provided?
+      type: condition
+      iscommand: false
+      description: ''
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "3"
+      "yes":
+      - "2"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: inputs.file_hash
+            iscontext: true
+          right:
+            value: {}
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: inputs.file_path
+            iscontext: true
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: inputs.endpoint_id
+            iscontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 190
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {},
+    "paper": {
+      "dimensions": {
+        "height": 745,
+        "width": 380,
+        "x": 50,
+        "y": 50
+      }
+    }
+  }
+inputs:
+- key: file_hash
+  value: {}
+  required: true
+  description: The file’s hash. Must be a valid SHA256 hash.
+  playbookInputQuery:
+- key: file_path
+  value: {}
+  required: true
+  description: The path for the selected file.
+  playbookInputQuery:
+- key: endpoint_id
+  value: {}
+  required: true
+  description: The file's location in the system.
+  playbookInputQuery:
+outputs: []
```

Comment:
no outputs?

Reply:
there is no output from the commands in this playbook because it is not defined for this automation.